generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -136,32 +140,22 @@ Scenario: Running a 'shorterThan' request alongside a contradicting matchingRege
        | foo    |
        | null   |
 
-@ignore #issue 294
 Scenario: Running a 'shorterThan' request alongside a non-contradicting containingRegex constraint should be successful
      Given foo is shorter than 3
-       And foo is containing regex /[🍩]{2}/
-     Then the following data should be included in what is generated:
-       | foo    |
-       | null   |
-       | "🍩🍩" |
-
-@ignore #issue 294
-Scenario: Running a 'shorterThan' request alongside a non-contradicting matchingRegex constraint should generate null
-     Given foo is shorter than 1
-       And foo is matching regex /[💾]{2}/
-      Then the following data should be generated:
-        | foo    |
-        | null   |
+       And foo is containing regex /[Æ]{2}/
+       And the generator can generate at most 2 rows
+     Then the following data should be generated:
+       | foo  |
+       | null |
+       | "ÆÆ" |
 
-@ignore #issue 294
 Scenario: Running a 'shorterThan' request alongside a contradicting containingRegex constraint should generate null
      Given foo is shorter than 1
-       And foo is containing regex /[🍩]{2}/
+       And foo is containing regex /[Ū]{2}/
       Then the following data should be generated:
         | foo    |
         | null   |
```

Comment:
Can it produce a string of length 0? so `""`

Reply:
No, because any string it creates has to contain regex /[Ū]{2}/ which it cannot do unless the length of the string is 2 or more.

Reply:
Ah okay, great thanks :)